processor/parse/flavored/callout.js

```diff
@@ -1,9 +1,6 @@
-// @note: We'd like to allow any emoji to match, but included in the emoji
-// character set is [#*0-9].
-// https://www.unicode.org/Public/13.0.0/ucd/emoji/emoji-data.txt
-const rgx =
-  // eslint-disable-next-line unicorn/no-unsafe-regex
-  /^> ?(\u00a9|\u00ae|[\u2000-\u3300]|[\u{1f000}-\u{1fbff}])(?: {0,}(.+))?\n((?:>(?: .*)?\n)*)/u;
+const emojiRegex = require('emoji-regex');
+
+const rgx = new RegExp(`^> ?(${emojiRegex().source})(?: +(.+)?)?\n((?:>(?: .*)?\n)*)`);
```

Comment:
@kellyjosephprice @trishaprile is `.source` a thing you can do on a `RegExp` instance? TIL

Reply:
[yup](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source) (we also just found out about it)!

Reply:
![giphy](https://user-images.githubusercontent.com/33762/180105197-97bfd3ad-6bc6-4a60-adf9-f9b15aa74667.gif)